Podfile

```diff
@@ -2,7 +2,7 @@ platform :ios, '9.0'
 use_frameworks!
 
 def shared_pods
-    pod 'Mapbox-iOS-SDK', '~> 4.1.1'
+    pod 'Mapbox-iOS-SDK', '~> 4.2.0'
```

Comment:
Can you drop back to 4.1.1? While we do want to update this, I think it should be done in separate PR.